app/test/features/chat/custom_input.dart

```diff
@@ -170,19 +216,21 @@ void main() {
       (tester) async {
         await tester.pumpWidget(
           ProviderScope(
-            overrides: overrides,
+            overrides: [
+              sdkProvider.overrideWith((ref) => MockActerSdk()),
+              ...overrides,
```

Comment:
nice! but do we really need to mock the ActerSDK itself? Isn't it enough to mock a higher level provider instead?

Reply:
The log points out to the last trace of sdk which is chained with other providers in the tree. The lib missing error only seems to resolve with this override not the other ones...